hal/armv8m/nrf/91/peripherals.h

```diff
@@ -0,0 +1,98 @@
+/*
+ * Phoenix-RTOS
+ *
+ * plo - operating system loader
+ *
+ * Peripherals definitions for armv8m33-nrf9160
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Damian Loewnau
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#ifndef _PERIPHERALS_H_
+#define _PERIPHERALS_H_
+
+#include <board_config.h>
+
+/* Periperals configuration */
+
+/* Based on INTLINESNUM value (ICTR cpu register) */
+#define SIZE_INTERRUPTS 256
+
+/* TIMER */
+/* We are using timer0, but it can be simply changed */
```

Comment:
This comment can become easily out-of-sync with the code. Better to mention valid values here instead

Reply:
That's right, done.